tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
People should rather have their git/ide settings correct. It isn't just a diff hindrance... it makes it harder to track down when specific lines are changed (via git blame).